docs/workspace-sync.html.md

```diff
@@ -222,6 +222,19 @@ variables:
     environmentVariable: true
 ```
 
+### Apply an SSH key to the Workspace (optional)
+
+To apply an SSH key to the workspace, specify `sshKeyID` in the Workspace Custom Resource. The SSH key ID can be found in the [Terraform Cloud API](https://www.terraform.io/docs/cloud/api/ssh-keys.html#list-ssh-keys).
```

Comment:
I think it would help if we included the Terraform Cloud documentation on how to add the SSH keys! Link here: https://www.terraform.io/docs/cloud/workspaces/ssh-keys.html

Reply:
Thanks, I like that idea. Added!